pom.xml

```diff
@@ -2479,10 +2465,7 @@
         <fasterxml.version>${fasterxml.spark3.version}</fasterxml.version>
         <fasterxml.jackson.databind.version>${fasterxml.spark3.version}</fasterxml.jackson.databind.version>
         <fasterxml.jackson.module.scala.version>${fasterxml.spark3.version}</fasterxml.jackson.module.scala.version>
-        <fasterxml.jackson.dataformat.yaml.version>${fasterxml.spark3.version}
```

Comment:
Why is dataformat.yaml being removed?

Reply:
dataformat.yaml is still there (see line 2468). Only  `<fasterxml.jackson.datatype.jsr310.module/>` is removed.